auth/auth0_manager.go

```diff
@@ -94,35 +96,30 @@ func (a *Auth0Manager) Authenticate(username, password string) (bool, error) {
 		Scope:      "openid",
 	}
 
-	if err := a.client.Post("/oauth/ro", req, nil); err != nil {
-		if err, ok := err.(*rclient.ResponseError); ok && (err.Response.StatusCode == 401 || err.Response.StatusCode == 429) {
-			cachedStatus.penalty += time.Second
-			if cachedStatus.penalty > maxPenalty {
-				cachedStatus.penalty = maxPenalty
-			}
-
-			if err.Response.StatusCode == 429 {
-				if a.throttleMultiplier > 0 {
-					a.throttleMultiplier = a.throttleMultiplier * 2
-				} else {
-					a.throttleMultiplier = 1
+	for backoff := time.Duration(0); true; backoff += time.Millisecond * 500 {
```

Comment:
For safety, lets add parens around this part `backoff += (time.Millisecond * 500)`